docs/draft/CMS/Publish_Content_Tutorial.adoc

```diff
@@ -223,27 +223,27 @@ We are going to create a new Content Model for the Content Type just created in
 ** {blank}
 +
 
-Go inside App Builder -> CMS -> Content Models and click on the ADD button
+Go inside App Builder -> CMS -> Content Template and click on the ADD button
 
 
 Fill the fields with these values:
 
 * {blank}
 +
 
-In the dropdown Type select the “_Conference_ ” Content Type, just created in step 1. Then click on the SET button. Keep in mind that in order for the content assist to work properly user must always select the Content Type
+In the dropdown Type select the “_Conference_ ” Content Type, just created in step 1. Then click on the SET button. Keep in mind that in order for the content assist to work properly, the user must always select the Content Type
 
 * {blank}
 +
 
-In the field *Code* insert the desired unique ID for the new Content Model: e.g. “_11112_”
+In the field *Code* insert the desired unique ID for the new Content Type: e.g. “_11112_”
```

Comment:
@dale1391 This is supposed to say Template I assume. Please updated. "type" is something else